.github/workflows/codeql.yml

```
@@ -0,0 +1,60 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.txt'
+      - '**/*.yaml'
+      - '**/*.json'
+      - '**/node_modules'
+      - '**/*.test.js'
+      - '**/*.test.ts'
```

Can be simplified to:

```suggestion
    paths:
      - 'dist/**'
      - 'src/**'
```